bin/NodeFSDriver.js

```diff
@@ -0,0 +1,29 @@
+const fs = require('fs');
+const path = require('path');
+const {promisify} = require('util');
+
+module.exports = class NodeFSDriver {
+
+  constructor() {}
+
+  static readFileAsync(path, encoding = 'utf8') {
+    return promisify(fs.readFile).call(null, path, encoding)
+  }
+
+  static writeDirectory(projectRelativePath) {
```

Comment:
The `NodeFSDriver` class, rather than an instantiated object, is injected into `FileSystem`, so `static` here fits that approach, but that approach in itself seems weird now. Will change to pass an object instantiated from `NodeFSDriver`, and make `NodeFSDriver`'s methods instance methods.